sofl/game.py

```
@@ -152,6 +156,62 @@ def remove_game(self) -> None:
         # The variable is the title of the game
         self.create_toast(_("{} removed").format(self.name), "remove")
 
+    def uninstall_game(self) -> None:
+        """Uninstall the game by removing its root directory after confirmation"""
+        # Check if the game is from online-fix
+        if "online-fix" not in self.source:
+            self.log_and_toast(_("Cannot uninstall non-online-fix games"))
```

Ruff (0.8.2)

Undefined name `_`

(F821)